docs/design/api/storage.rst

```diff
@@ -0,0 +1,324 @@
+Storage API
+###########
+
+For this examples let's say:
+
+- Buyers logs with Firefox Account Bearer Tokens
+- Seller apps logs with Hawk
+- Payment App logs with Basic Auth
+
+
+As a user, I want to retrieve my information about my purchases
+===============================================================
+
+.. code-block:: http
+
+    GET /buckets/mozilla/collections/payments/records
```

Comment:
I think this could be shortened to ``GET /buckets/mozilla/collections/purchases`` since in all cases it will be a collection of purchased products. But maybe you're using ``payments`` as a namespace? The URL would work either way.